app/models/route.rb

```diff
@@ -3,4 +3,17 @@ class Route < ApplicationRecord
   has_many :bus_stops_routes
   has_many :bus_stops, through: :bus_stops_routes
   default_scope { order :number }
+
+  def stops_by_sequence(route_id, direction)
+    BusStop
```

Comment:
why is `BusStop` all alone on this line? It's not so readable in my opinion. I'd suggest to at least have `BusStop.joins(:bus_stops_routes)` for the first line of this query.